migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs

```diff
@@ -59,6 +59,17 @@ impl SqlRenderer for PostgresFlavour {
                 })
                 .collect();
 
+            if stmts.len() > 1 {
+                let warning = indoc::indoc! {
+                    r#"-- This migration adds more than one variant into an enum.
+                       -- With PostgreSQL versions 11 and earlier, this is not possible
+                       -- in one migration. Either split them into separate files, or run
+                       -- them manually in SQL console."#
```

Comment:
I don't think we should recommend doing the change manually in the SQL console. You would end up with a database that is inconsistent with migrations and the migrations could not be applied.

Since the user is using Migrate, we should a fix using Migrate only, IMO.